server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -176,38 +193,47 @@ export class ScribeLambdaFactory
 				this.enableWholeSummaryUpload,
 			);
 			latestSummary = await summaryReader.readLastSummary();
+			latestSummaryCheckpoint = latestSummary.scribe
+				? JSON.parse(latestSummary.scribe)
+				: undefined;
+			latestDbCheckpoint = (await this.checkpointService.restoreFromCheckpoint(
+				documentId,
+				tenantId,
+				"scribe",
+				document,
+			)) as IScribe;
 		} catch (error) {
-			const errorMessage = "Scribe lambda creation failed.";
-			context.log?.error(`${errorMessage} Exception: ${inspect(error)}`, { messageMetaData });
-			Lumberjack.error(errorMessage, lumberProperties, error);
-			await this.sendLambdaStartResult(tenantId, documentId, {
-				lambdaName: LambdaName.Scribe,
-				success: false,
-			});
-			scribeSessionMetric?.error("Scribe lambda creation failed", error);
-
+			await failCreation(error);
 			throw error;
 		}
 
+		// Mongodb casts undefined as null so we are checking both to be safe.
+		// Also, if the document is new, it will not have a local scribe checkpoint.
```

Comment:
Just a clarification here, there will not be a local scribe checkpoint if the document is new, or if it is the start of a new active session (meaning the first client of a new session joins) since we clear the scribe checkpoint after receiving a `noClient` op

Reply:
Updated this comment to be more specific to the "new document" case. Thanks for pointing this out!